recipes/microprofile/all/patches/cmake.patch

```diff
@@ -0,0 +1,132 @@
+--- /dev/null
++++ b/CMakeLists.txt
```

Comment:
Did you send this cmake file to the upstream as a PR? I know that project abandoned, but still is valid.

Reply:
I haven't yet. It will probably need some changes, but I will open a PR to the upstream with this CMake file.

Reply:
For reference, here is the upstream PR I opened https://github.com/jonasmr/microprofile/pull/71.